sky/utils/cli_utils/status_utils.py

```diff
@@ -298,57 +311,71 @@ def _get_resources(cluster_status):
     return resources_str
 
 
-def _get_resources_for_cost_report(cluster_status):
-    launched_nodes = cluster_status['num_nodes']
-    launched_resources = cluster_status['resources']
-
-    launched_resource_str = str(launched_resources)
-    resources_str = (f'{launched_nodes}x '
-                     f'{launched_resource_str}')
-
-    return resources_str
-
-
-def _get_zone(cluster_status):
-    zone_str = cluster_status['handle'].launched_resources.zone
+def _get_zone(cluster_record: _ClusterRecord) -> str:
+    zone_str = cluster_record['handle'].launched_resources.zone
     if zone_str is None:
         zone_str = '-'
     return zone_str
 
 
-def _get_price(cluster_status):
-    launched_nodes = cluster_status['num_nodes']
-    launched_resources = cluster_status['resources']
-
-    hourly_cost = (launched_resources.get_cost(3600) * launched_nodes)
-    price_str = f'$ {hourly_cost:.3f}'
-    return price_str
-
-
-def _get_autostop(cluster_status):
+def _get_autostop(cluster_record: _ClusterRecord) -> str:
     autostop_str = ''
     separtion = ''
-    if cluster_status['autostop'] >= 0:
+    if cluster_record['autostop'] >= 0:
         # TODO(zhwu): check the status of the autostop cluster.
-        autostop_str = str(cluster_status['autostop']) + 'm'
+        autostop_str = str(cluster_record['autostop']) + 'm'
         separtion = ' '
 
-    if cluster_status['to_down']:
+    if cluster_record['to_down']:
         autostop_str += f'{separtion}(down)'
     if autostop_str == '':
         autostop_str = '-'
     return autostop_str
 
 
-def get_cost_report(cluster_status):
-    cost = cluster_status['total_cost']
+def _is_pending_autostop(cluster_record: _ClusterRecord) -> bool:
+    return _get_autostop(cluster_record) != '-' and _get_status(
+        cluster_record).value != 'STOPPED'
```

Comment:
Should we directly compare the enum, instead of the `.value`?

Reply:
Thanks, done.